CONTRIBUTING.md

```diff
@@ -243,12 +243,8 @@ Any user-visible change should have a `Changes` entry:
 
 - using the label "`*`" if it breaks existing programs, "`-`" otherwise
 
-- with the issue number `PR#{N}` if from mantis, `GPR#{N}` if from github
-  (several numbers separated by commas can be used)
-
-- maintaining the order: each section lists Mantis PRs first in ascending
-  numerical order, followed by Github PRs in ascending numerical order,
-  followed by changes that are not related to a PR.
+- with all relevant issue and PR numbers `#{N}`, in ascending numerical order
```

Comment:
It's unclear if you mean that numbers should be sorted within the entry (they should) and/or entries should be sorted within the section (they should)

Reply:
I pushed an amended rewording.